Template CI.yaml

```diff
@@ -0,0 +1,333 @@
+name: Template-CI-3-$(Build.BuildId)
```

Comment:
Take the 3 out of here too.

Reply:
Renamed the "Template-CI-3-$(Build.BuildId)" to Template-CI-$(Build.BuildId)